### ISSUE TITLE
Add support for WebDAV

### ISSUE THREAD

**Issue by mrpdaemon** [mrpdaemon/encdroid#50]

**Comment by ashh87** [comment]
WebDAV appears to be implemented here:

https://play.google.com/store/apps/details?id=org.mrpdaemon.android.encdroidmc&hl=en

However, I can't find the source, which I think should be available under GPL(?) Anyway, that app seems to have a few features that could be ported back to the original, has this been discussed before?

**Comment by mrpdaemon** [comment]
Yes, I'm aware of the fork - I have corresponded with the author in the past. He did send me source code of his work whenever I asked for it (no GPL issues) but he doesn't seem very interested in collaborating. The fork is from a quite old point in Encdroid's development so his WebDAV work might not be easily merge-able.
